typescript/web/src/components/labeling-tool/image-navigation/image-navigation-tool.fixtures.ts

```diff
@@ -0,0 +1,50 @@
+import { pick } from "lodash";
+import { MATCH_ANY_PARAMETERS } from "wildcard-mock-link";
+import {
+  GetAllImagesOfADatasetQuery,
+  GetAllImagesOfADatasetQueryVariables,
+} from "../../../graphql-types/GetAllImagesOfADatasetQuery";
+import { GET_ALL_IMAGES_OF_A_DATASET_QUERY } from "../../../hooks/use-images-navigation";
+import {
+  ApolloMockResponse,
+  ApolloMockResponses,
+} from "../../../utils/tests/apollo-mock";
+import {
+  DEEP_DATASET_WITH_IMAGES_DATA,
+  BASIC_DATASET_DATA,
+} from "../../../utils/tests/data.fixtures";
+
+const DATASET_SLUG_DATA = {
+  [BASIC_DATASET_DATA.slug]: BASIC_DATASET_DATA,
+  [DEEP_DATASET_WITH_IMAGES_DATA.slug]: DEEP_DATASET_WITH_IMAGES_DATA,
+};
+
+export const GET_ALL_IMAGES_OF_A_DATASET_MOCK: ApolloMockResponse<
+  GetAllImagesOfADatasetQuery,
+  GetAllImagesOfADatasetQueryVariables
+> = {
+  request: {
+    query: GET_ALL_IMAGES_OF_A_DATASET_QUERY,
+    variables: MATCH_ANY_PARAMETERS,
+  },
+  result: jest.fn((variables) => ({
```

Comment:
Again here, this should be moved in function(s) so that the mock feels less complex to read